app/actions/inreach_client/client.py

```diff
@@ -0,0 +1,125 @@
+import json
+import os
+from typing import List, Optional
+from urllib.parse import urljoin
+import httpx
+from gundi_core.schemas.v2 import InReachIPCMessage
+from .errors import (
+    InReachClientError,
+    InReachAuthenticationError,
+    InReachServiceUnreachable,
+    InReachInternalError, exceptions_by_inreach_code,
+)
+
+
+class InReachClient:
+    DEFAULT_API_URL = os.getenv("INREACH_API_URL", "https://eur-enterprise.inreach.garmin.com")
```

Comment:
From past experience, the value `https://explore.garmin.com` is probably the best for default fallback. 
```suggestion
    DEFAULT_API_URL = os.getenv("INREACH_API_URL", "https://explore.garmin.com")
```

Reply:
Thanks, updated it